xbmc/video/VideoInfoTag.cpp

```diff
@@ -738,6 +747,8 @@ void CVideoInfoTag::Serialize(CVariant& value) const
   value["tag"] = m_tags;
   value["videoversion"] = m_typeVideoVersion;
   value["videoversionid"] = m_idVideoVersion;
+  value["hasvideoversions"] = m_hasVideoVersions;
```

Comment:
Needs minor bump on Json API as far as I remember

Reply:
Not sure, something else needs to be done to make the info available via the API, as far as I remember. Putting it here is just a prerequisite for the API, without further consequences. 

Maybe @DaveTBlake  can shed some light on this.

Intention of this PR is not to make the new stuff available via the API.

Reply:
^^ That.  There is much more to do to make it available via the API then just serialising it.  This PR doesn't touch the API so no need to bump the version.

Reply:
Ok